.gitignore

```diff
@@ -9,4 +9,5 @@ coverage
 dist
 lib
 !test/**/node_modules
-.vscode/
\ No newline at end of file
+.vscode/
+.idea/
```

Comment:
IDE specific ignores like .vscode and .idea probably belong in your own, machine local ignore file, or you'll end up with a huge list in a public project.